pkg/kernel/tools/mechutils/mechutils.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2020-2021 Cisco and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+
+// Package mechutils provides utilities for conververtin kernel.Mechanism to various things
+package mechutils
+
+import (
+	"fmt"
+	"net/url"
+	"runtime"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/kernel"
+	"github.com/pkg/errors"
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
```

Comment:
Most of these methods logically duplicates methods from `tools/nshandle`, so I am not actually sure if we need to move it.

Reply:
@Bolodya1997 I just merged modules and let me if it's looking ok.